src/Ganeti/HTools/Backend/Luxi.hs

```diff
@@ -308,15 +310,16 @@ readData master =
 parseData :: (Result JSValue, Result JSValue, Result JSValue, Result JSValue)
           -> Result ClusterData
 parseData (groups, nodes, instances, cinfo) = do
+  (ctags, cpol, master, hypervisor) <- cinfo >>= getClusterData
```

Comment:
Good point, it is a relict of my previous attempt to put the mapping inside of getNodes, but it turned out to be less invasive here.